diff --git a/dist/node/chunks/dep-BWSbWtLw.js b/dist/node/chunks/dep-BWSbWtLw.js
index d7bddd3da64618901cbaa96f64dd05c2a0daa934..8f9581c41d9d1dab9bb2cece405446db7ccd9805 100644
--- a/dist/node/chunks/dep-BWSbWtLw.js
+++ b/dist/node/chunks/dep-BWSbWtLw.js
@@ -51125,11 +51125,20 @@ function esbuildOutputFromId(outputs, id, cacheDirOutputPath) {
   if (output) {
     return output;
   }
+	const testPaths = []
   for (const [key, value] of Object.entries(outputs)) {
-    if (normalizePath$3(path$n.relative(cwd, key)) === normalizedOutputPath) {
+		const testPath = normalizePath$3(path$n.relative(cwd, key))
+		testPaths.push(testPath)
+    if (testPath === normalizedOutputPath) {
       return value;
     }
   }
+	console.trace(`esbuildOutputFromId cannot locate output: ${id}
+cwd: ${cwd}
+cacheDirOutputPath: ${cacheDirOutputPath}
+normalizedOutputPath: ${normalizedOutputPath}
+testPaths: ${testPaths.join('\n')}
+`);
 }
 async function extractExportsData(filePath, config, ssr) {
   await init;
